ChangeLog.md

```diff
@@ -1,5 +1,275 @@
 # Changelog for haskell-language-server
 
+## 1.1.0
+
+Haskell Language Server 1.1.0 has finally come! Many thanks to all contributors -- since the last release, we have merged over 100 PRs!
+As always, there are many internal bug fixes and performance improvements in ghcide. Apart from that,
+
+* Wingman gets a comprehensive evolution, thanks to @isovector for this epic work!
```

Comment:
Here are the highlights in my eyes, from most to least interesting:

* Wingman actions can now be bound to editor hotkeys
* Experimental support for "jump to next unsolved hole"
* Improved layout algorithm --- don't reflow instances, or break do-blocks
* Wingman can now deal with GADTs, rank-n types and pattern synonyms
* Wingman now respects user-written bindings on the left side of the equals sign
* Significantly more-natural synthesized code when dealing with newtypes, infix operators, records and strings
* Improved user experience --- less waiting, and friendly errors for when things go wrong

Reply:
I've added them. Thanks!